src/screens/scanning/ManualVoucherScreen.tsx

```diff
@@ -74,12 +75,27 @@ function ManualVoucherScreen() {
           </FieldContainer>
           <FieldContainer>
             <InputTitleText>Amount</InputTitleText>
-            <InputField
-              onChange={onChangeVoucherAmount}
+            <CurrencyInput
               value={voucherAmount}
-              placeholder="Enter Amount"
-              validate={validateVoucherAmount}
-              keyboardType="decimal-pad"
+              onChangeValue={onChangeVoucherAmount}
+              renderTextInput={props => (
+                <TextInput
```

Comment:
Nit: I would add a TODO comment in the code here to remember that this needs to be refactored.